### PR TITLE
Update localstack to 3.0.0

### PR DESCRIPTION
**Pull request by slang25**
This version includes support for the SQS JSON protocol

### CODE REVIEW

File: tests/JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj [hunk 1]
```diff
@@ -12,8 +12,8 @@
     <ProjectReference Include="..\JustSaying.TestingFramework\JustSaying.TestingFramework.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.SimpleNotificationService" />
-    <PackageReference Include="AWSSDK.SQS" />
+    <PackageReference Include="AWSSDK.SimpleNotificationService" VersionOverride="3.7.300.5" />
+    <PackageReference Include="AWSSDK.SQS" VersionOverride="3.7.300.5" />
```
Comment (martincostello) [inline tests/JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj:16]: TIL - does this work with dependabot?

File: tests/JustSaying.UnitTests/Messaging/Channels/Fakes/FakeAmazonSqs.cs [hunk 2]
```diff
@@ -48,6 +49,11 @@ public Task<string> AuthorizeS3ToSendMessageAsync(string queueUrl, string bucket
         return Task.FromResult(new AddPermissionResponse());
     }
 
+    public Task<CancelMessageMoveTaskResponse> CancelMessageMoveTaskAsync(CancelMessageMoveTaskRequest request, CancellationToken cancellationToken = new CancellationToken())
```
Comment (martincostello) [inline tests/JustSaying.UnitTests/Messaging/Channels/Fakes/FakeAmazonSqs.cs:52]: We should change all these optional parameters to use the simplified syntax:

```suggestion
    public Task<CancelMessageMoveTaskResponse> CancelMessageMoveTaskAsync(CancelMessageMoveTaskRequest request, CancellationToken cancellationToken = default)
```

File: tests/JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj (hunk 1 above)
Comment (slang25) [reply tests/JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj:16]: Good question, I don't know, would be good to find out. There is also a feature that I can't find where you define versions in groups, and you can say in the package reference "I want the version from group A", can't remember how that works, so doing this for now.

Comment (martincostello) [reply tests/JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj:16]: I guess we can merge it and if it messes stuff up we can try something else.

**Review by martincostello — APPROVED** [review]

**Pull request merged**